README.md

```diff
@@ -1292,8 +1303,9 @@ The configuration node contains two properties for applying security to your bot
 
 <img src="images/TelegramBotSecurity.png" title="Applying security" width="500" />
 
-**Fig. 29:** Security configuration in the bot configuration node
+**Fig. 31:** Security configuration in the bot configuration node
 
+**Note**: The *Users* in the security configuration are define via their usernames. These usernames are configured in the telegram app via the settings dialog. Configured usernames typically begin with a '@' in the app. In the *Users* field, no '@' is used.
```

Comment:
are defined

Reply:
Typo added to the pull request few minutes ago.